src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -30,15 +30,15 @@ private class FindLiteralsProgressAdapter : IStreamingFindLiteralReferencesProgr
                 _definition = definition;
             }
 
-            public async Task OnReferenceFoundAsync(Document document, TextSpan span)
+            public async Task OnReferenceFoundAsync(Document document, TextSpan span, CancellationToken cancellationToken)
             {
                 var documentSpan = await ClassifiedSpansAndHighlightSpan.GetClassifiedDocumentSpanAsync(
                     document, span, _context.CancellationToken).ConfigureAwait(false);
```

Comment:
We can't remove "IFindUsagesContext", it's the API we pass to 3rd parties to provide results.   It contains the cancellation token they're supposed to respect.   This follows the pattern we use all the time for cancellation operations with results that are pushed back.  For example CompletionContext and CodeFixContext